specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json

```diff
@@ -0,0 +1,177 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/failover": {
+      "post": {
+        "tags": [
+          "ManagedInstances"
+        ],
+        "description": "Failovers a managed instance.",
+        "operationId": "ManagedInstances_Failover",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
```

Comment:
nit: You don't have to re-define your own standard parameters. Please reference the ones under common-type\resource-management\v1\types.json. Since there are many instances in the current api-version folder, you may defer to cleanup time. But should at least standardize to single common definition in your RP as the previous comment indicated.

Reply:
Done.